lib/event_sourcery.rb

```diff
@@ -39,10 +59,16 @@ def self.config
     @config ||= Config.new
   end
 
+  # Logger object used by EventSourcery. Set via `configure`.
+  #
+  # @see Config.logger
   def self.logger
     config.logger
   end
 
+  # Registry of all ESPs
```

Comment:
Maybe "Registry of all Event Stream Processors"? It might be not clear what ESP is.